serveProduction.js

```diff
@@ -197,9 +197,8 @@ const processChat = async (channel_id, msg) => {
 
     const sanitizedChat = sanitizeChat(msg)
 
-    const request = "http://www.purgomalum.com/service/json?text=" + sanitizedChat
-    const playerNameRequest = "http://www.purgomalum.com/service/json?text=" + decodedMario.getPlayername()
-
+    const request = "http://www.purgomalum.com/service/json?text=" + encodeURIComponent(sanitizedChat)
+    const playerNameRequest = "http://www.purgomalum.com/service/json?text=" + encodeURIComponent(decodedMario.getPlayername())
```

Comment:
Also start encoding the URI for the request.

Reply:
Because & is used for seperating query params in URI. It should be encoded

Reply:
also allow rainbow mario. but this is a good thig tho, would like this in base game.

Reply:
What